tests/logprob/test_transforms.py

```diff
@@ -807,16 +807,36 @@ def test_discrete_rv_multinary_transform_fails():
         conditional_logp({y_rv: y_rv.clone()})
 
 
-@pytest.mark.xfail(reason="Check not implemented yet")
-def test_invalid_broadcasted_transform_rv_fails():
+@pytest.mark.filterwarnings("error")  # Fail if unexpected warning is issued
+@pytest.mark.parametrize("implicit_broadcast", (True, False))
+def test_broadcasted_transform_rv(implicit_broadcast):
```

Comment:
Correct me if I'm wrong here: logprob inference for broadcasted variables work only if they are 1) pure RVs and 2) affine-transformed pure RVs?

Reply:
No. It should work for anything that is explicitly under `BroadcastTo` or implicitly in one of the supported binary Elemwise (Add/ Mul / Pow...)